src/vip_client/classes/VipCI.py

```diff
@@ -162,7 +164,10 @@ def init(cls, vip_key="VIP_API_KEY", girder_key="GIRDER_API_KEY", verbose=True,
         # Restore the verbose state
         cls._VERBOSE = verbose
         # Instantiate a Girder client
-        cls._girder_client = girder_client.GirderClient(apiUrl=cls._GIRDER_PORTAL)
+        cls._girder_client = girder_client.GirderClient(apiUrl=girder_api_url)
+        # Set the Girder ID prefix
+        cls._GIRDER_ID_PREFIX = girder_id_prefix if girder_id_prefix is not None else cls._GIRDER_ID_PREFIX
+        cls._GIRDER_PORTAL = girder_api_url if girder_api_url is not None else cls._GIRDER_PORTAL
```

Comment:
should not the girder client be created with _GIRDER_PORTAL after it is created ?